### PR TITLE
add 'eukaryote detection and abundance analysis'

### LINKED ISSUE #392: NTR: Eukaryote detection and abundance analysis
For new term requests, please provide the following information:
**Term Label** Eukaryote detection and abundance analysis

**Synonyms** NA

**Definition** Abundance of eukaryotic species detected using the CORRAL method [[1](https://www.biorxiv.org/content/10.1101/2022.03.09.483664v1)]

**Suggested Parent Term** Whole metagenome sequencing assay https://www.ebi.ac.uk/ols/ontologies/eupath/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FOBI_0002623

**Examples** NA

**Contributor** NA

**Requester** MicrobiomeDB

### PR DESCRIPTION
**Pull request by johnwjudkins**
Closes #392.

### CODE REVIEW

**Comment by johnwjudkins** [comment]
Since the definition refers to CORALE software, I changed the label.

File: src/ontology/eupath_dev.owl
Comment (asizemore) [inline src/ontology/eupath_dev.owl:6792]:
```suggestion
        <obo:IAO_0000115>A data transformation that uses data from a whole metagenome sequencing assay and CORRAL (for Clustering of Reference Alignments) software to specify taxa of Eukaryota in a specimen.</obo:IAO_0000115>
```

File: src/ontology/eupath_dev.owl
Comment (asizemore) [inline src/ontology/eupath_dev.owl:6796]:
```suggestion
        <rdfs:label>eukaryote detection and abundance analysis using CORRAL software</rdfs:label>
```

**Review by asizemore — CHANGES_REQUESTED** [review]
In general looks great to me! But worth getting approval from Dan on wording specifics before merging.

**Comment by asizemore** [comment]
Sorry just realized i left the same comments here and in the issue. But i just spoke with Dan and he also thought it's great after the CORALE -> CORRAL changes! Thanks so much @johnwjudkins !

**Comment by johnwjudkins** [comment]
I see the most recent version of the article reads "CORRAL". I changed the label & definition.

File: src/ontology/eupath_dev.owl
Comment (asizemore) [inline src/ontology/eupath_dev.owl:6792]:
```suggestion
        <obo:IAO_0000115>A data transformation that uses data from a whole metagenome sequencing assay and CORRAL (for Clustering Of Related Reference ALignments) software to specify taxa of Eukaryota in a specimen.</obo:IAO_0000115>
```

**Review by asizemore — APPROVED** [review]
Thanks @johnwjudkins ! 
I apologize I missed a related change from the CORALE -> CORRAL switch in the paper. I put that in as a requested change but otherwise looks great!

File: src/ontology/eupath_dev.owl
Comment (asizemore) [reply src/ontology/eupath_dev.owl:6792]: (for reference, the above change was just copied from the [paper](https://www.biorxiv.org/content/10.1101/2022.03.09.483664v2))

**Review by zhengj2007 — APPROVED** [review]

**Pull request merged**
